lib/collections/schemas/helpers.js

```diff
@@ -55,3 +56,25 @@ export function schemaIdAutoValue() {
   }
   return this.unset();
 }
+
+/**
+ * shopDefaulCountry
+ * @summary used for schema injection autoValue
```

Comment:
I hadn't seen `@summary` or `@example` used before - would you mind adding those to the ongoing discussion in docs? https://github.com/reactioncommerce/reaction-docs/issues/275

Reply:
`shopDefaultCountry` - minor typo!